database/languages/c-programming.json

```diff
@@ -1 +1,7 @@
-[]
\ No newline at end of file
+[ {
+    "name": "FreeCodeCamp",
```

Comment:
[Nitpick] We have a few sources from FreecodeCamp, so I think it's best to use the course's name to make it more distinct from the others.
```suggestion
    "name": "FreeCodeCamp",
```

Reply:
Hey @CBID2 
I have changed the source name from "FreeCodeCamp" to Course-name
![2023-06-11](https://github.com/rupali-codes/LinksHub/assets/102748498/376ca9f3-4e44-4bb2-821b-c196190e0a06)
.
Kindly review it :)